### ISSUE TITLE
False position on variable-declaration & exception

### ISSUE THREAD

**Issue by fforjan** [palantir/tslint#390]
My code is something like this : 
``` typescript
    try {
      activeDocument.Content = $("textarea#umlDocument").val();
      JUMLY.eval($("textarea#umlDocument"), {
        into: $("div#renderer")
      });
    } catch (_error ) {
      var ex: any = _error;
      $("div#status").text(ex);
   }
```

I got this: 
variable-declaration: '_error' to have a typedef

Which is wrong as according to TS1013, Catch clause parameter cannot have a type annotation

**Comment by adidahiya** [comment]
which version of tslint are you using?

**Comment by adrienkohlbecker** [comment]
I'm seeing this as well:

```
try {
  console.log("test");
} catch (err) {
  console.error("test");
}
```

Gives `test.ts[3, 13]: expected variable-declaration: 'err' to have a typedef`
Using tsling 2.2.0-beta

**Comment by adidahiya** [comment]
this was fixed by #422, which will be included in v2.3.0
